ci-operator/step-registry/ipi/deprovision/nutanix/preload-image/ipi-deprovision-nutanix-preload-image-ref.yaml

```diff
@@ -0,0 +1,14 @@
+ref:
+  as: ipi-deprovision-nutanix-preload-image
+  from_image:
+    namespace: origin
+    name: centos
+    tag: '8'
```

Comment:
I think centos 8 reach EOL. For newly add stuff, we'd better avoid that.

Reply:
Is it OK to remove line  ` tag: '8'` ?

Reply:
or you can use `from: cli`